Backend/prisma/schema.prisma

```diff
@@ -25,10 +24,17 @@ model suggestions {
   content        String
```

Comment:
remove title for suggestion, there is only the content

Reply:
I might add it later so we will leave it for now